mscore/pagesettings.cpp

```diff
@@ -41,38 +41,40 @@ PageSettings::PageSettings(QWidget* parent)
 
-      mmUnit = true;      // should be made a global configuration item
-
-      if (mmUnit)
-            mmButton->setChecked(true);
-      else
-            inchButton->setChecked(true);
-
       MuseScore::restoreGeometry(this);
 
-      for (int i = 0; i < QPageSize::LastPageSize; ++i)
-            pageGroup->addItem(QPageSize::name(QPageSize::PageSizeId(i)), i);
-
-      connect(mmButton,             SIGNAL(clicked()),            SLOT(mmClicked()));
-      connect(inchButton,           SIGNAL(clicked()),            SLOT(inchClicked()));
-      connect(buttonApply,          SIGNAL(clicked()),            SLOT(apply()));
+      typesList->addItem("Metric");
+      typesList->addItem("Imperial");
+      typesList->addItem("Other");
+
+	for (int i = 0; i <= QPageSize::Cicero; ++i)
+	      unitsList->addItem(QString("%1 (%2)").arg(pageUnits[i].name())
+                                                 .arg(pageUnits[i].suffix()));
+      
+      ///!!!widget naming is inconsistent across modules in MuseScore.
+      ///!!!I am following a mix of conventions and leaving some names unchanged
+      connect(buttonReset,          SIGNAL(clicked()),            SLOT(resetToDefault()));
       connect(buttonApplyToAllParts,SIGNAL(clicked()),            SLOT(applyToAllParts()));
+      connect(buttonApply,          SIGNAL(clicked()),            SLOT(apply()));
       connect(buttonOk,             SIGNAL(clicked()),            SLOT(ok()));
-      connect(portraitButton,       SIGNAL(clicked()),            SLOT(orientationClicked()));
-      connect(landscapeButton,      SIGNAL(clicked()),            SLOT(orientationClicked()));
+      connect(portraitButton,       SIGNAL(toggled(bool)),        SLOT(orientationToggled(bool)));
+      connect(landscapeButton,      SIGNAL(toggled(bool)),        SLOT(orientationToggled(bool)));
       connect(twosided,             SIGNAL(toggled(bool)),        SLOT(twosidedToggled(bool)));
-      connect(pageHeight,           SIGNAL(valueChanged(double)), SLOT(pageHeightChanged(double)));
-      connect(pageWidth,            SIGNAL(valueChanged(double)), SLOT(pageWidthChanged(double)));
-      connect(oddPageTopMargin,     SIGNAL(valueChanged(double)), SLOT(otmChanged(double)));
+      ///!!!currentIndexChanged only fires when the index changes, activated always fires
+      connect(typesList,            SIGNAL(currentIndexChanged(int)), SLOT(typeChanged(int)));
+      connect(sizesList,            SIGNAL(currentIndexChanged(int)), SLOT(sizeChanged(int)));
+      connect(unitsList,            SIGNAL(currentIndexChanged(int)), SLOT(unitsChanged()));
+      connect(pageWidth,            SIGNAL(valueChanged(double)), SLOT(widthChanged(double)));
+      connect(pageHeight,           SIGNAL(valueChanged(double)), SLOT(heightChanged(double)));
+      connect(spatiumEntry,         SIGNAL(valueChanged(double)), SLOT(spatiumChanged(double)));
+	connect(oddPageTopMargin,     SIGNAL(valueChanged(double)), SLOT(otmChanged(double)));
```

Comment:
There is a tab at the beginning of this line instead of 6 spaces.

Reply:
fixed - how does that even happen?  I have never had the insert tabs option set.